auth/src/main/java/com/firebase/ui/auth/ui/phone/PhoneActivity.java

```diff
@@ -406,4 +409,14 @@ private SubmitConfirmationCodeFragment getSubmitConfirmationCodeFragment() {
         return (SubmitConfirmationCodeFragment) getSupportFragmentManager().findFragmentByTag
                 (SubmitConfirmationCodeFragment.TAG);
     }
+
+    private void setUpTermsOfServiceFooter() {
```

Comment:
Yeah, I'm definitely thinking we need a utility, this is numeros quatros I've seen. 😆

Reply:
:sweat_smile: